lib/sales_engine.rb

```diff
@@ -282,16 +287,18 @@ def find_most_revenue_items(x)
     item_hash = items_revenue.group_by{ |n| n[1] }
     item_revenue = []
     for n in 1..10000
-
       unless item_hash[n].nil?
         item_revenue << [item_hash[n].collect{ |n| n[0] }.reduce(:+), item_hash[n].flatten[1]]
       end
     end
 
-    item_ids = item_revenue.sort[-x..-1].collect {|i| i[1]}
+    item_ids = item_revenue.sort[-x..-1].collect { |i| i[1] }
     items = item_ids.map do |item_id|
       item_repository.find_by_item_id(item_id)
     end
     return items.reverse
   end
+
+  def find_most_popular_items(x)
```

Comment:
Unused method argument - `x`. If it's necessary, use `_` or `_x` as an argument name to indicate that it won't be used. You can also write as `find_most_popular_items(*)` if you want the method to accept any arguments but don't care about them.